### ISSUE TITLE
h5py installed via conda is not found by pip

### ISSUE THREAD

**Issue by penenkel** [ContinuumIO/anaconda-issues#320]
EDIT:
SYSTEM: Windows 7
CONDA: v3.9.1  win-32
PYTHON: 2.7

After installing h5py via conda, pip cannot find it (it can be imported & used within a python session, but pip cannot find it).  This causes trouble with further pip installed packages that require h5py. One effect is that pip tries to install h5py again (usually failing because e.g. MS compilers are not installed). One can work around that by using `--no-deps`, but apparently any package that uses pkg_resources introspection will also fail with `pkg_resources.DistributionNotFound: The 'H5py' distribution was not found and is required by [...]`

Creating a new environment to demonstrate:

```
PS H:\_Software\Python> conda create -n test python=2 h5py
Fetching package metadata: ....
Solving package specifications: .
Package plan for installation in environment H:\_Software\Python\python_32\test:

The following NEW packages will be INSTALLED:

    h5py:       2.5.0-np19py27_0
    numpy:      1.9.2-py27_0
    pip:        6.1.1-py27_0
    python:     2.7.9-1
    setuptools: 15.2-py27_0
    six:        1.9.0-py27_0

Proceed ([y]/n)? y

Linking packages ...
[      COMPLETE      ] |#################################################| 100%

PS H:\_Software\Python> conda list -n test
# packages in environment at H:\_Software\Python\python_32\test:
#
h5py                      2.5.0                np19py27_0
numpy                     1.9.2                    py27_0
pip                       6.1.1                    py27_0
python                    2.7.9                         1
setuptools                15.2                     py27_0
six                       1.9.0                    py27_0
```

And then with the environment activated:

```
PS H:\_workspace> pip list
numpy (1.9.2)
pip (6.1.1)
setuptools (15.2)
six (1.9.0)
```

EDIT:
Compared to the windows installer for h5py the .egg folder seems to be missing in the conda version.

**Comment by ZhangAustin** [comment]
+1
same issue here.

**Comment by nehaljwani** [comment]
This has been fixed in `h5py=2.5.0=np19py27_1`
```powershell
(320) C:\Users\nwani>conda list h5py
# packages in environment at C:\Users\nwani\Anaconda2\envs\320:
#
h5py                      2.5.0                np19py27_1

(320) C:\Users\nwani>pip freeze
h5py==2.5.0
numpy==1.9.3
six==1.10.0
```
